v2/apierror/api_error.go

```diff
@@ -0,0 +1,236 @@
+// Copyright 2021, Google Inc.
```

Comment:
nit: filename should just be `apierror.go`

Reply:
done!